.github/workflows/build.yml

```diff
@@ -102,17 +126,23 @@ jobs:
 
   deploy:
     if: github.event_name != 'pull_request'
-    needs: build
+    needs: [smoke-tests, build]
     runs-on: ubuntu-latest
     steps:
+      -
+        name: Checkout code
+        uses: actions/checkout@v3
       -
         name: Get Binaries built in previous jobs
         uses: actions/download-artifact@v3.0.0
+        id: download
         with:
           name: tmp-binary-storage
+          path: ftl-builds/
```

Comment:
@yubiuser do you recall the reason for adding this `path:` part?

This happened... 
![image](https://user-images.githubusercontent.com/1998970/188211728-a0b1ea5f-21f3-4cbc-b052-a8521713c429.png)

Reply:
Wasn't there a change made to make sure that the parent directory always existed?